src/parsing/css-url-image-value-parsing.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//     You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(internal) {
+
+  function consumeURLImageValue(string) {
+    // need to remove 'url("' and '")'
+    var temp = string.substring(5);
```

Comment:
You can use the other parsing consume* methods to clarify this and also fail correctly if the string isn't in the form url(stuff).

Take a look at the other parsers and see how they use the consume* libraries, for example the handling for "matrix(1,2,3,4,5,6)".
 https://github.com/css-typed-om/typed-om/blob/master/src/parsing/css-matrix-parsing.js